tests/virtualization/xen/hotplugging.pm

```diff
@@ -22,37 +22,73 @@ use version_utils;
 sub run_test {
     my ($self) = @_;
     my $hypervisor = get_var('HYPERVISOR') // '127.0.0.1';
+    my ($sles_running_version, $sles_running_sp) = get_sles_release("/etc/os-release");
+
+    my %mac = ();
+    record_info "Virtual network", "Adding virtual network interface";
+    foreach my $guest (keys %xen::guests) {
+        my $persistent_config_option = '';
+        my $interface_model_option   = '';
+        if (get_var('VIRT_AUTOTEST')) {
```

Comment:
These two should be there to initialize these two variables:
my $persistent_config_option = '';
my $interface_model_option   = '';
Because if conditions are not met they could become undefined. Although any reference to them will not throw exception, "uninitialized variable" will be reported in log file